sync-core/src/main/java/com/cloudant/mazha/CouchURIHelper.java

```diff
@@ -61,6 +63,17 @@ public URI changesUri(Map<String, Object> query) {
                 this.rootUriString,
                 "_changes"
         );
+
+        //lets find the since parameter
+        if(query.containsKey("since")){
```

Comment:
Not really, encoding it here means only the helper needs to know about encoding, where as if we place it at the call site for the method we have the potential for the implementation detail about sequence numbers to leak out more